cluster/kubernetes/update_test.go

```diff
@@ -149,7 +149,6 @@ spec:
     metadata:
       labels:
         name: fluxy
-        version: master-a000001
```

Comment:
This removal isn't meaning-preserving as advertised in the commit message - incidental cleanup perhaps?

Reply:
Yes, I expect I removed this as part of kneading the tests into shape, and forgot to record the fact in the commit message. Sorry!

kubeyaml doesn't attempt to update this label -- it was a vestigial part of ReplicationController support, iirc.